test/finance/VestingWallet.behavior.js

```diff
@@ -22,35 +24,51 @@ function shouldBehaveLikeVesting(beneficiary) {
     }
   });
 
-  it('execute vesting schedule', async function () {
-    const [release, ...args] = this.token ? ['release(address)', this.token.address] : ['release()'];
+  describe('execute vesting schedule', async function () {
+    beforeEach(async function () {
+      [this.release, ...this.args] = this.token ? ['release(address)', this.token.address] : ['release()'];
+    });
 
-    let released = web3.utils.toBN(0);
-    const before = await this.getBalance(beneficiary);
+    it('releases a linearly vested schedule', async function () {
+      let released = web3.utils.toBN(0);
+      const before = await this.getBalance(beneficiary);
+      const releaser = await this.mock.releaser();
 
-    {
-      const receipt = await this.mock.methods[release](...args);
+      {
+        // Allows gas price to be 0 so no ETH is spent in the transaction.
+        await setNextBlockBaseFeePerGas(0);
```

Comment:
I guess you added that so that the gas cost don't interfer with measuring the balances movement ... but we don't check the balances movements.

In [test/finance/VestingWallet.test.js](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4502/files#diff-3632407a8b9c87a3131e1d83a32f0397eeba2af61927b061825f67bdba757d1d), we may want to add somthing to the `this.checkRelease` for Eth Vesting

Reply:
This no longer applies because the `onlyReleaser` commit was reverted.